### PR TITLE
Add current and expected jsons with transformer for edx.team.searched

### PR DESCRIPTION
**Pull request by HamzaFarooqArbisoft**
**Trello Link:** _https://trello.com/c/NvpxudQH/118-teams-related-event-edxteamsearched_

**Description:** _When a user performs a search for teams from the topic view under the Teams page of the courseware, the server emits an edx.team.searched event.
_

**Checks before merge:**

- [x] Reviewed
- [x] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/edx.team.searched.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "object": {
+    "extensions": {
+      "course_id": "course-v1:edx+CS0110+2014_T",
+      "number_of_results": 0,
+      "org_id": "edx",
+      "search_text": "hello",
+      "topic_id": "Topic1ID"
+    },
+    "id": "http://localhost:18000/courses/course-v1:edx+CS0110+2014_T/teams/",
+    "type": "Frame"
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/expected/edx.team.searched.json]: What is the reason behind using `Frame` here?

Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/tests/expected/edx.team.searched.json]: done

File: openedx/features/caliper_tracking/tests/current/edx.team.searched.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "event_source": "server",
+  "event_type": "edx.team.searched",
+  "host": "8c3b43aa7b1b",
+  "ip": "172.18.0.1",
+  "name": "edx.team.searched",
+  "page": null,
+  "referer": "http://localhost:18000/courses/course-v1:edx+CS0110+2014_T/teams/",
+  "session": "8b5aaa43369a0d612918005f1a65f27f",
+  "time": "2018-12-26T12:57:21.978410+00:00",
+  "username": "edx"
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/current/edx.team.searched.json]: Please use a more descriptive username

File: openedx/features/caliper_tracking/transformers/__init__.py [hunk 3]
```diff
@@ -65,7 +65,6 @@
     textbook_pdf_page_navigated,
     textbook_pdf_page_scrolled,
     textbook_pdf_search_executed,
-    textbook_pdf_page_navigated,
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/__init__.py]: Why is this being removed?

Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/transformers/__init__.py]: its a duplicate entry. that's why.

File: openedx/features/caliper_tracking/tests/current/edx.team.searched.json (hunk 2 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/tests/current/edx.team.searched.json]: done

**Pull request merged**